### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by palmersp** [thomaslindstrom/color-picker#59]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.177.0
**System**: Microsoft Windows 8.1
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.4.4

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /C:/Users/Spencer/.atom/packages/color-picker/lib/ColorPicker-view.coffee:302

TypeError: undefined is not a function
  at C:\Users\Spencer\.atom\packages\color-picker\lib\ColorPicker-view.coffee:209:56
  at Object.Emitter.emit (C:\Users\Spencer\.atom\packages\color-picker\lib\ColorPicker-view.coffee:172:31)
  at C:\Users\Spencer\.atom\packages\color-picker\lib\ColorPicker-view.coffee:185:65

```

### Commands

```
     -5:29.8 timecop:view (atom-text-editor.editor.autocomplete-active.is-focused)
     -3:26.9 command-palette:toggle (button.btn.btn-default.icon.icon-link)
  2x -3:09.0 markdown-preview:toggle (atom-text-editor.editor)
     -0:48.4 command-palette:toggle (atom-text-editor.editor)
     -0:32.2 editor-stats:toggle (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.6
autoclose-html, v0.15.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.1.0
color-picker, v1.4.4
editor-stats, v0.17.0
emmet, vundefined
highlight-line, v0.9.3
minimap, v4.1.0
monokai, v0.12.0
set-syntax, v0.2.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by arcticicestudio** [comment]
Got the same error when clicking somewhere in opened [svg-preview](https://atom.io/packages/svg-preview) window.

**Affected packages**
`color-picker` 1.4.4
`svg-preview` 0.5.1

**Comment by NM-** [comment]
Just got a similar error when I was in settings and clicked cod view[Enter steps to reproduce below: 

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.182.0
**System**: Mac OS X 10.10.2
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.4.4

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/neco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209

TypeError: undefined is not a function
  at /Users/neco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209:56
  at Object.Emitter.emit (/Users/neco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:172:31)
  at /Users/neco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:185:65

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v1.4.4
linter, v0.11.1
minimap, v4.3.1

# Dev
No dev packages
```

**Comment by michaelachrisco** [comment]
Same Issue:
```
At /Users/michaelchrisco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209

TypeError: undefined is not a function
  at /Users/michaelchrisco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209:56
  at Object.Emitter.emit (/Users/michaelchrisco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:172:31)
  at /Users/michaelchrisco/.atom/packages/color-picker/lib/ColorPicker-view.coffee:185:65
```

**Comment by edgebal** [comment]
[Enter steps to reproduce below:]

1. Clicked on [editor-stats](https://github.com/atom/editor-stats) panel

Note: I have vim-mode plugin enabled, but it also happens if those plugins are disabled.

**Atom Version**: 0.200.0
**System**: Mac OS X 10.10.3
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.7.0


### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/emi/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209

TypeError: undefined is not a function
  at /Users/emi/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209:56
  at Object.Emitter.emit (/Users/emi/.atom/packages/color-picker/lib/ColorPicker-view.coffee:172:31)
  at /Users/emi/.atom/packages/color-picker/lib/ColorPicker-view.coffee:185:65

```

### Commands

```
     -0:32 core:close (atom-text-editor.editor.vim-mode.visual-mode.is-focused)
     -0:32 blur (atom-text-editor.editor.vim-mode.visual-mode.is-focused)
     -0:31 core:close (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  7x -0:31 blur (atom-text-editor.editor.vim-mode.command-mode.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "git-diff"
    ],
    "themes": [
      "seti-ui",
      "seti-syntax"
    ],
    "projectHome": "/Users/emi/Majestic/htdocs"
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v1.7.0
docblockr, v0.7.1
editor-stats, v0.17.0
editorconfig, v1.0.0
emmet, v2.3.10
ex-mode, v0.4.1
flex-tool-bar, v0.2.1
git-log, v0.3.0
git-tab-status, v1.9.2
jumpy, v2.0.6
linter, v0.12.6
linter-php, v0.0.15
seti-syntax, v0.4.0
seti-ui, v0.7.1
tool-bar, v0.1.1
tool-bar-main, v0.0.8
vim-mode, v0.48.0

# Dev
No dev packages
```

**Comment by ekonstantinidis** [comment]
Same here!

    TypeError: undefined is not a function
      at /Users/Manos/.atom/packages/color-picker/lib/ColorPicker-view.coffee:209:56
      at Object.Emitter.emit (/Users/Manos/.atom/packages/color-picker/lib/ColorPicker-view.coffee:172:31)
      at /Users/Manos/.atom/packages/color-picker/lib/ColorPicker-view.coffee:185:65

Happends when I'm trying to search and I press the match case button. 
![screen shot 2015-05-22 at 16 46 45](https://cloud.githubusercontent.com/assets/6333409/7774318/3e4609a2-00a2-11e5-8ea5-c75f5a2ef549.png)

**Comment by thomaslindstrom** [comment]
This should be fixed in the upcoming `v2.0.0`. If this error is still present in `v2.0.0`, please re-open this issue.
